README.md

```diff
@@ -1,27 +1,38 @@
-## Kopano Identity Management
+## LibreGraph Identity Management
 
-The kidm daemon provides a LDAP server which is easy to configure, does not have external dependencies and is tailored to work perfectly with other Kopano software.
+The LibreGraph Identity Management provides a LDAP server, which is easy to configure, does not have external dependencies and is tailored to work perfectly with other LibreGraph software.
 
-The goal is that everyone who does not already have or needs an LDAP server, uses kidmd.
+The goal is that everyone who does not already have or needs an LDAP server, uses IDM.
 
-Thus, kidmd is a (read-only) drop in replacement for an existing LDAP server and does provide an LDAP interface if none is there already.
+Thus, IDM is a (currently read-only) drop in replacement for an existing LDAP server and does provide an LDAP interface if none is there already. IDM uses hard coded indexes and supports LDAP search, bind and unbind operations.
 
-Kidmd supports LDAP search, bind and unbind.
+### Running idmd from a source build
 
+Until packages and containers for more environments are available it is the easiest to just create a local build of `idmd`. For this just run `make`.
+
+IDM uses a mixture of environment variables and parameters for configuration and needs to be at least passed a the location of an individual ldif file or a directory containing multiple ldif files.
+
+```bash
+$ ./idmd serve --ldif-main ./export.ldif
+INFO[0000] LDAP listener started                         listen_addr="127.0.0.1:10389"
+INFO[0000] ready
+```
+
+When packaging up IDM it is recommended to invoke idm through its [binscript](https://github.com/libregraph/idm/blob/master/scripts/libregraph-idmd.binscript) to simplify execution.
```

Comment:
I don't think that its required to state that here - the readme should be as easy and short as possible imo.